docs/madr/decisions/073-spiffe-compliance.md

```diff
@@ -153,7 +157,921 @@ Future releases
+              op: add
+              value:
+                name: spire-agent-socket
+                hostPath:
+                  path: /run/spire/sockets
+                  type: Directory
+            - path: spec.template.spec.containers.kuma-sidecar.volumeMounts.spire-agent-socket
+              op: add
+              value:
+                name: spire-agent-socket
+                mountPath: /run/spire/sockets/
+                readOnly: true          
+```
+
+#### Define active identity provider
+
+Previously, defining the active identity for a group of services was done through a single, perhaps less flexible, mechanism. Now, the `MeshIdentity` resource, leveraging its `selector`, offers a more granular and controlled approach to activate specific configurations.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+metadata:
+  name: old-identity
+  namespace: kuma-system # only in system namespace
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    dataplane:
+      matchLabels: {} # select all
+....
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+metadata:
+  name: new-identity
+  namespace: kuma-system # only in system namespace
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    dataplane:
+      matchLabels:
+        app: demo-app
+```
+
+The `MeshIdentity` resource’s selector field allows selection based on dataplane labels. Users can define it to apply to either all dataplanes or a specific subset.
+If there are multiple (X) identities selecting the same dataplane, the control plane will resolve the conflict by choosing one based on the following order of precedence:
+
+1. Selector specificity – the identity with most specific selector is preferred.
+2. Lexicographical order – if selector sizes are equal, the identity with the lexicographically smallest name is chosen.
+
+**Examples**
+
+1. Select only specific dataplane
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+...
+spec:
+  selector:
+    dataplane:
+      matchLabels:
+        app: demo-app
+```
+
+2. Select all dataplanes
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+...
+spec:
+  selector:
+    dataplane:
+      matchLabels: {}
+```
+
+3. Select nothing
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+...
+spec:
+  selector:
+    dataplane: {}
+```
+
+or
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+...
+spec:
+  selector:
+```
+
+or
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshIdentity
+...
+spec:
+...
+```
+
+It seems to be consistant with LabelsMatcher from kubernetes api https://github.com/kubernetes/apimachinery/blob/v0.33.2/pkg/apis/meta/v1/helpers.go#L36-L43.
+
+```golang
+func LabelSelectorAsSelector(ps *LabelSelector) (labels.Selector, error) {
+	if ps == nil {
+		return labels.Nothing(), nil
+	}
+	if len(ps.MatchLabels)+len(ps.MatchExpressions) == 0 {
+		return labels.Everything(), nil
+	}
+```
+
+#### What would the process of switching backends look like?
```

Comment:
what's the value of keeping the same fields? They mention `backends` which are not things that exist anymore in this new world